data/Pakistan/Assembly/ep-popolo-v1.0.json

```diff
@@ -4420,6 +4416,37 @@
       "patronymic_name": "w/o Sohail Sarfraz"
     },
     {
+      "gender": "male",
+      "id": "47223c4f-9ca3-4d33-928a-258adee23f2d",
+      "identifiers": [
+        {
+          "identifier": "Q33076529",
+          "scheme": "wikidata"
+        }
+      ],
+      "image": "http://www.na.gov.pk/uploads/images/NA-260%20sEngr.%20sMuhammad%20sUsman%20sBadini.jpg",
+      "images": [
+        {
+          "url": "http://www.na.gov.pk/uploads/images/NA-260%20sEngr.%20sMuhammad%20sUsman%20sBadini.jpg"
+        }
+      ],
+      "links": [
+        {
+          "note": "Wikipedia (en)",
+          "url": "https://en.wikipedia.org/wiki/Usman_Badini"
+        }
+      ],
+      "name": "Engineer Mohammad Usman Badaini",
```

Comment:
Hmmm. We probably want to start trapping prefixes like this. It's a pre-existing problem, so it doesn't need to be done in this change, but we'd probably want to fix it before letting any new people in with the same problem, especially as this is now a well-solved issue with a solution that can copied easily from other scrapers. It also seems like the sort of thing that's going to get in the way of our Wikidata prompts, so that might also accelerate us wanting to fix it.

Reply:
I've opened https://github.com/everypolitician/everypolitician-data/issues/53128 to capture the problem with the name prefixes.